tbx/static_src/sass/config/_variables.scss

```diff
@@ -35,6 +35,7 @@ $color--mid-grey: #282e30;
     --color--accent-four: #{$color--purple};
     --color--accent-five: #{$color--white};
     --color--decoration: #{$color--grey-50};
+    --letter-motif: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="131" height="65" viewBox="0 0 131 65" style="background-color:white"><g><path d="M42.736 5.7715C42.5176 5.10569 41.5291 5.11241 41.276 5.77735C39.8026 9.75854 37.3683 13.3882 35.4475 14.7155L35.3759 14.3315C30.3916 -10.3914 8.75224 -20.284 1.27801 -22.9593C0.56863 -23.2251 -0.0866055 -22.4414 0.351579 -21.8605C6.36903 -14.1036 6.23971 -5.69242 0.284686 3.57819C-0.857281 5.29079 -2.05639 7.45542 -3.02772 9.57781C-3.02772 9.57781 -4.21677 12.0496 -5.16359 15.3151L-5.27059 15.2467C-9.13408 11.4605 -10.7876 6.38163 -11.457 3.54818C-11.6298 2.87393 -12.5887 2.78686 -12.9033 3.37491C-14.3555 6.20448 -16.6975 11.8476 -16.1729 17.4448C-14.8765 30.977 -3.91163 43.0255 11.2519 43.966C11.9671 44.01 12.341 43.2343 11.8276 42.7557C10.2021 41.1148 8.93462 38.6129 8.29849 35.1995C6.82745 27.3058 13.7048 13.6264 13.7048 13.6264C13.7048 13.6264 25.0509 23.9268 26.522 31.8205C27.166 35.2766 26.8845 38.0657 25.9588 40.1796C25.6977 40.8019 26.326 41.3917 26.9321 41.1027C40.3736 35.079 44.7134 21.6936 44.3241 14.0407C44.1674 10.6708 43.3295 7.69201 42.736 5.7715Z" fill="%237045FF"/></g><g style="mix-blend-mode:multiply"><path d="M-35.1294 26.8277C-35.4119 26.0114 -34.5018 25.3236 -33.7882 25.8078C-29.5543 28.7367 -24.6951 30.6405 -21.9744 30.6126L-22.1872 30.18C-35.5367 2.06174 -22.889 -22.9913 -17.9831 -30.9062C-17.5261 -31.6692 -16.3583 -31.3492 -16.3386 -30.463C-16.2284 -18.5118 -10.0153 -10.2277 2.1552 -5.18072C4.44186 -4.27779 7.10837 -2.96492 9.53571 -1.53408C9.53571 -1.53408 12.4156 0.0916725 15.6488 2.67798L15.6972 2.53466C16.4916 -3.9496 14.3257 -10.1683 12.8856 -13.4598C12.5553 -14.2526 13.3703 -15.0129 14.0845 -14.6484C17.4646 -12.8514 23.6981 -8.87788 27.2734 -2.93624C35.8915 11.4485 34.5808 31.1487 21.3767 42.7384C20.7536 43.2847 19.8492 42.7751 19.9725 41.9378C20.272 39.162 19.6199 35.7804 17.7291 31.9347C13.3565 23.0415 -2.85316 14.2527 -2.85316 14.2527C-2.85316 14.2527 -5.77962 32.4801 -1.40703 41.3733C0.507448 45.2671 2.78614 47.8464 5.16564 49.3008C5.85562 49.737 5.70754 50.7657 4.94324 50.9037C-11.7303 54.3494 -25.4032 44.0705 -30.5919 36.1771C-32.8903 32.7116 -34.2814 29.157 -35.1294 26.8277Z" fill="%23FC8698"/></g></svg>');
```

Comment:
The mix-blend mode needs to come out here to avoid the inaccessible colours. It's a bit hard to tell from testing - is this based on the same one I used as test.svg? If so i wonder if it needs cropping closer to the edge of the image - it had a lot of white.

Reply:
It was - I think it needs a bit of white space to get the top left to bottom right transition working but let's see what Ben says before we adjust it